src/View/Form/EntityContext.php

```diff
@@ -454,6 +454,29 @@ public function getRequiredMessage($field)
         return null;
     }
 
+    /**
+     * Get field length from validation
+     *
+     * @param string $field The dot separated path to the field you want to check.
+     * @return int|null
+     */
+    public function getMaxLength($field)
+    {
+        $parts = explode('.', $field);
+        $validator = $this->_getValidator($parts);
+        $fieldName = array_pop($parts);
+        if (!$validator->hasField($fieldName)) {
+            return null;
+        }
+        foreach ($validator->field($fieldName)->rules() as $rule) {
+            if ($rule->get('rule') === 'maxLength') {
+                return isset($rule->get('pass')[0]) ? $rule->get('pass')[0] : null;
```

Comment:
Is the `isset()` check necessary?  You can't use max length validator without specifying the max length argument.